### ISSUE TITLE
Set Up Data Extraction Tools

### ISSUE THREAD

**Issue by grayJiaaoLi** [amosproj/amos2024ss08-cloud-native-llm#28]
## User story
1. As a developer
2. I need to set up tools that can automatically extract data from our collected CNCF resources
3. So that the data can feed into our Q&A pipeline

## Acceptance criteria
* Pull data from collected CNCF resources
* Store data in a format that is ready for processing
* Log the errors when the problems occur

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by dominic0df** [comment]
What is your aim of this issue? We already have data extraction implemented in https://github.com/amosproj/amos2024ss08-cloud-native-llm/issues/16
